modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -163,3 +163,15 @@ box when creating the `/boot/efi` partition.
 WARNING: Choosing to reformat `/boot/efi` will likely result in the inability
 to boot any other operating systems that were previously installed.  Be sure that
 yau have backed up any important data before using this workaround.
+
+Unable to enter a toolbox due to permissions errors
+----------------------------------------------------
+https://github.com/containers/libpod/issues/3187
+
+With certain versions of `libpod`, trying to enter a toolbox will result in
```

Comment:
I'd change `libpod` to `podman`, since users won't be able to do something like `rpm -q libpod` and get a version.

Reply:
Good point, changed it. I'll do the rebase onto current master now.